ckanext/datastore/set_permissions.sql

```diff
@@ -55,17 +55,20 @@ CREATE OR REPLACE VIEW "_table_metadata" AS
         substr(md5(dependee.relname || COALESCE(dependent.relname, '')), 0, 17) AS "_id",
```

Comment:
Can you summarize what the changes in the permissions mean here? It's a bit hard to follow.

Will all existing DataStore users need to re-run this against their existing database?